### PR TITLE
[coverage] VRRP, L2TP, HSRP, RIP, & Radius tests

### PR DESCRIPTION
**Pull request by guedou**
Here are some fixes and tests for VRRP, L2TP, HSRP, RIP, & Radius.

### CODE REVIEW

**Comment by gpotter2** [comment]
@guedou Good job with that...
As a note: you may have noticed that on AppVeyor, codecov seems to act a bit weirdly:
```
codecov
'git' is not recognized as an internal or external command,
operable program or batch file.
'git' is not recognized as an internal or external command,
operable program or batch file.
'git' is not recognized as an internal or external command,
operable program or batch file.
'hg' is not recognized as an internal or external command,
operable program or batch file.
'hg' is not recognized as an internal or external command,
operable program or batch file.
'find' is not recognized as an internal or external command,
operable program or batch file.
      _____          _
     / ____|        | |
    | |     ___   __| | ___  ___ _____   __
    | |    / _ \ / _  |/ _ \/ __/ _ \ \ / /
    | |___| (_) | (_| |  __/ (_| (_) \ V /
     \_____\___/ \____|\___|\___\___/ \_/
                                    v2.0.5
==> Detecting CI provider
    AppVeyor Detected
    Error running `git log -1 --pretty=%B`: None
==> Preparing upload
    Error running `cd C:\projects\scapy && git ls-files`: None
    Error running `git ls-files`: None
    Error running `cd C:\projects\scapy && hg locate`: None
    Error running `hg locate`: None
```

**Comment by guedou** [comment]
@gpotter2 the error message is present since we introduced codecov on Windows. The git command is not found in the PATH variable. I did not take the time to fix it as codecov works.

**Comment by p-l-** [comment]
This PR seems odd: does it include code from other PRs? Anyway it has conflicts.

**Comment by guedou** [comment]
Sorry, I messed up the initial `git branch`. It is now rebased and fixed.

FYI, the test was failing because the HSRP layer overrides the IP.dst. Scapy used my local IP address, and the test was failing everywhere else than locally.

File: scapy/layers/hsrp.py [hunk 1]
```diff
@@ -67,7 +67,7 @@ class HSRPmd5(Packet):
         ByteEnumField("algo", 0, {1: "MD5"}),
         ByteField("padding", 0x00),
         XShortField("flags", 0x00),
-        IPField("sourceip", None),
+        IPField("sourceip", "127.0.0.1"),
```
Comment (p-l-) [inline scapy/layers/hsrp.py]: Can you merge #466? That would make the current `None` default value working and more relevant that `"127.0.0.1"`.

Comment (guedou) [inline scapy/layers/hsrp.py]: Do you want me to change IPField to SourceIPField here ?

Comment (p-l-) [inline scapy/layers/hsrp.py]: Yes, if that's supposed to be a source IP address.

**Comment by guedou** [comment]
@p-l- can you restart the AppVeyor build ? It seems that it failed due to a networking issue. Thanks.

File: scapy/layers/hsrp.py (hunk 1 above)
Comment (p-l-) [inline scapy/layers/hsrp.py]: Now if you rebase against master, you can change this to `SourceIPField("sourceip", None)`

Comment (p-l-) [inline scapy/layers/hsrp.py]: Or I can do that in my PR about `SourceIPField`.

**Comment by guedou** [comment]
Thanks !

**Pull request merged**